django/src/rdwatch/models/site_observation.py

```diff
@@ -11,6 +12,7 @@
 
 
 class SiteObservation(models.Model):
+    created = CreationDateTimeField()
```

Comment:
In my other comment about the timestamp for the `siteEvaluation` being like a 'created' date.  Each `SiteObservation` has a `siteeval_id` which would link it to a `siteEvaluation` and that has a timestamp that is created when it is uploaded.